x-pack/legacy/plugins/maps/public/layers/styles/vector/components/color/color_map_select.js

```diff
@@ -98,16 +130,39 @@ export class ColorMapSelect extends Component {
         : '';
     }
 
+    let toggle;
+    if (this.props.showColorMapTypeToggle) {
+      toggle = <EuiFlexItem grow={false}>{this._renderColorMapToggle()}</EuiFlexItem>;
+    } else {
+      toggle = <Fragment />;
+    }
+
+    return (
+      <EuiFlexGroup gutterSize={'none'}>
+        {toggle}
+        <EuiFlexItem>
+          <EuiSuperSelect
+            compressed
+            options={colorMapOptionsWithCustom}
+            onChange={this._onColorMapSelect}
+            valueOfSelected={valueOfSelected}
+            hasDividers={true}
+          />
+        </EuiFlexItem>
+      </EuiFlexGroup>
+    );
+  }
+
+  render() {
     return (
       <Fragment>
-        <EuiSuperSelect
-          options={colorMapOptionsWithCustom}
-          onChange={this._onColorMapSelect}
-          valueOfSelected={valueOfSelected}
-          hasDividers={true}
-          compressed
-        />
-        {this._renderColorStopsInput()}
+        {this._renderColorMapSelections()}
+
+        <EuiSpacer size="s" />
+
+        <EuiFlexGroup>
+          <EuiFlexItem>{this._renderColorStopsInput()}</EuiFlexItem>
```

Comment:
`_renderColorMapSelections` and `_renderColorStopsInput` are not consistent. `_renderColorMapSelections` includes FlexGroup while `_renderColorStopsInput` does not. I would recommend pushing FlexGroup wrapper into `_renderColorStopsInput` to be more consistent.

Reply:
Can you still address this comment?